modules/dataquery/jsx/react.app.js

```diff
@@ -181,17 +181,19 @@ class DataQueryApp extends Component {
       for (let i = 0; i < this.state.queryIDs[key].length; i += 1) {
         let curRequest;
         curRequest = Promise.resolve(
-          $.ajax(loris.BaseURL + '/AjaxHelper.php?Module=dataquery&script=GetDoc.php&DocID=' + encodeURIComponent(this.state.queryIDs[key][i])), {
-            data: {
-              DocID: this.state.queryIDs[key][i]
-            },
-            dataType: 'json'
-          }).then((value) => {
-          let queries = this.state.savedQueries;
-
-          queries[value._id] = value;
-          this.setState({savedQueries: queries});
-        });
+          fetch(
+            window.location.origin
+            + '/dataquery/Document?DocID='
```

Comment:
by convention our URLs are usually lowercase, and I think it would be a lot cleaner if the URL was `/dataquery/documents/xxxx` and not `/dataquery/document?DocID=xxxx`

Reply:
@driusan I tried changing the url to `dataquery/documents/xxx` but then `$values = $request->getQueryParams();` doesn't seem to work for getting the DocID from the GET request. I did change it to lowercase and to be `documents` instead of `document` with latest commit. @xlecours

Reply:
This isn't resolved. It should still be documents/xxx and not ?DocID=xxx, but after changing that the xxx isn't a query param, so it needs to be extracted from $request->getUri().

Reply:
@driusan I'm not really sure what you mean by documents/xxx. Do you mean just having the url like /documents/DocID=xxx. In any case let me know if the latest commit which is another way of adding params to a url isn't good either.

Reply:
No, I mean a document named "foo" should be accessed at `/documents/foo`. It isn't a parameter, it's the document name.

Reply:
Thanks @driusan for the clarification, that makes sense. I wasn't thinking like that lol.